source/fem-dem/gls_vans.cc

```diff
@@ -325,8 +325,14 @@ GLSVANSSolver<dim>::update_solution_and_constraints()
                                                 system_rhs_void_fraction);
 
   void_fraction_constraints.clear();
+
+  // reinitialize affine constraints with the correct size
```

Comment:
I would just say "reinitialize affine constraints". The second part sounds like there is something wrong prior to this, and you need to correct it.